src/content/docs/validator/external-chains/xrpl.mdx

```diff
@@ -14,6 +14,10 @@ Node setup guide for your XRP Ledger Testnet or Mainnet validator.
 - Ubuntu Linux 18.04+ (tested on 22.04) or Debian 10+
 - [Official Documentation](https://xrpl.org/docs/infrastructure/installation)
 
+<Callout emoji="💡">
+  Note: Verifiers are expected to run at least two months worth of history on their nodes. Please make sure you configure your node to stores two months of history instead of the [default two hours](https://xrpl.org/docs/infrastructure/installation/capacity-planning#disk-space).
```

Comment:
Did you run this through anyone on the eng team to confirm?

Reply:
yes confirmed with eng